### ISSUE TITLE
Hochregallager sinnvoll nutzen und einsortieren

### ISSUE THREAD

**Issue by maikf** [raumzeitlabor/rzl-tuwat#31]
Da ja das Hochregallager jetzt aufgeräumt ist, stellt sich die Frage,
wie wir vorgehen, dass nicht wieder unendlich viel Müll drin steht.

Mir schwebt vor die unteren Zeilen für die restlichen MitgliederIkeaBoxen zu nutzen,
die zZ verteilt im Raum und im "Textillager" stehen.

Generell fände ich es gut, nichts dauerhaft im Hochregallager zu lagern (Eventkram
fällt mir dazu ein), das ist besser in unserem Lager aufgehoben.

**Comment by Don42** [comment]
Die oberen Ebenen kann man denke ich schon für längerfristigere Lagerung nutzen.
Auch wenn mir aktuell nichts direkt einfällt was darunter fallen würde.

**Comment by else** [comment]
Solange länger*fristig* eine Frist beinhaltet...

**Comment by Don42** [comment]
Das stimmt. Eine Inventarisierung wäre vermutlich auch keine schlechte Idee. Dann könnte man Dinge mit "Instructed", "Give away" oder wie die grünen und gelben Aufkleber heißen einfacher finden.

**Comment by else** [comment]
Vielleicht helfen "intelligentere" Aufkleber auch weiter. Es gab vor kurzem eine Diskussion auf der Backspace ML. Rote Aufkleber (Owner only) z.B. sollten meiner Meinung nach immer mit einer Fristangabe verbunden sein (könnte dann ja verlängert werden, wenns weiter relevant ist).

**Comment by Don42** [comment]
Keine schlechte Idee. Könntest du ein Ticket aufmachen und die Diskussion verlinken oder zusammenfassen?

**Comment by Bfritz0815** [comment]
@else gibts für die Aufkleber auch Vorlagen oder so? Magst du das im Plenum vorstellen?

**Comment by uwap** [comment]
Gab es da noch eine weitere diskussion zu? Ansonsten würde ich das gerne im Mai oder Juni Plenum noch mal ansprechen.

**Comment by steeph-k** [comment]
Finde die Idee auch gut. Wer hat denn damals die Aufkleber gemacht/bestellt? Manche Sorten sind schon lange leer. Owner-Aufkleber dürften als nächstes leer gehen.

**Comment by uwap** [comment]
Angeblich ist das ja "zu viel Bürokratie". Dann lassen wir das Hochregallager halt zumüllen.

**Comment by tabascoeye** [comment]
Die Aufkleber hat @silsha über Stickma bestellt iirc.
Vorlagenmäßig haben wir das damals beim shackspace geklaut

**Comment by silsha** [comment]
Ja, die Aufkleber kommen von DeineStadtKlebt. Vorlagen gibt’s hier: https://github.com/raumzeitlabor/artwork/tree/master/sticker/Label

**Comment by uwap** [comment]
Wir haben ja seit einiger Zeit beschriftungen der Regalzeilen und Spalten (ist vermutlich noch niemandem aufgefallen). Diese könnten wir endlich mal nutzen um ein wenig Ordnung reinzubringen. Eventuell können wir einige Dauerkisten wie zB Agendadiplom oder MRMCD im Wiki genau so dokumentieren wie das mit der Tardis ist.
"Wo finde ich denn xyz vom Agendadiplom?"
"Das Wiki sagt im Hochregallager in D4"

Das wäre zumindest schon mal ein erster Schritt in richtung sinnvolle Sortierung.
